TicTacToeAppTests/ViewControllerSpec.swift

```diff
@@ -0,0 +1,101 @@
+import Quick
+import Nimble
+import TicTacToeApp
+
+class ViewControllerSpec: QuickSpec {
+  override func spec() {
+
+    describe("ViewController") {
+
+      it("disables button after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.enabled).to(beFalse())
+      }
+
+      it("changes button title after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.currentTitle).notTo(beNil())
+      }
+
+      it("changes button title to alternating player marks after each move") {
+        let controller = ViewController()
+        let firstMoveButton = UIButton()
+        let secondMoveButton = UIButton()
+        let thirdMoveButton = UIButton()
+
+        controller.makeMove(firstMoveButton)
+        controller.makeMove(secondMoveButton)
+        controller.makeMove(thirdMoveButton)
+
+        expect(firstMoveButton.currentTitle).to(equal("X"))
+        expect(secondMoveButton.currentTitle).to(equal("O"))
+        expect(thirdMoveButton.currentTitle).to(equal("X"))
+      }
+
+      it("removes board button titles when the game is reset") {
+        let controller = ViewController()
+        controller.boardButtons = [
```

Comment:
Sounds good. Thinking about it further, calling makeMove() entails checking if the game is over, which assumes the existence of 9 buttons - so it's likely that every test that calls that method depends on there being 9 buttons initialized. We'll need to break up the contexts and/or change the design before I can reduce the number of initialized buttons.